extra_sources/custom_memory_manager.c

```diff
@@ -293,22 +294,17 @@ void *pvPortReallocMicroROS( void *pv, size_t xWantedSize )
 	vTaskSuspendAll();
 
 	void * newmem = pvPortMallocMicroROS(xWantedSize);
+	if (newmem != NULL)
+	{
+		size_t count = getBlockSize(pv) - xHeapStructSize;
```

Comment:
Why are you substracting `xHeapStructSize` here? I guess that is not necessary.
```suggestion
		size_t count = getBlockSize(pv);
```

Reply:
I have checked it and that's correct.